front/lib/models/assistant/conversation.ts

```diff
@@ -86,6 +87,61 @@ Conversation.belongsTo(Workspace, {
   foreignKey: { name: "workspaceId", allowNull: false },
 });
 
+export class ConversationParticipant extends Model<
+  InferAttributes<ConversationParticipant>,
+  InferCreationAttributes<ConversationParticipant>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare action: ParticipantActionType;
+
+  declare conversationId: ForeignKey<Conversation["id"]>;
+  declare userId: ForeignKey<User["id"]>;
+}
+ConversationParticipant.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    action: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    modelName: "conversation_participant",
+    sequelize: front_sequelize,
+  }
```

Comment:
We probably want a unique index on conversationId userId and an index on userId ?

Reply:
I added indexes for both userId, and (userId, conversationId). 
Adding the unique would mean we won't be able to store multiple actions for the same couple user/convo, do we want that?

Reply:
I thiiink if it's called "Participant" ?